.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,36 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: BUG
+labels: bug
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior.
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**Desktop (please complete the following information):**
+ - OS: [e.g. iOS]
+ - Browser [e.g. chrome, safari]
+ - Fairlearn version [e.g. 0.4.4]
+ - Python version [e.g. 3.5.4]
+ - version of Fairlearn dependencies, i.e. ideally the output of `pip freeze`
```

Comment:
it's easier if you give people a way to extract this information. For instance, [we have](https://raw.githubusercontent.com/scikit-learn/scikit-learn/master/.github/ISSUE_TEMPLATE/bug_report.md):

```
Please run the following snippet and paste the output below.
For scikit-learn >= 0.20:
import sklearn; sklearn.show_versions()
For scikit-learn < 0.20:
import platform; print(platform.platform())
import sys; print("Python", sys.version)
import numpy; print("NumPy", numpy.__version__)
import scipy; print("SciPy", scipy.__version__)
import sklearn; print("Scikit-Learn", sklearn.__version__)
import imblearn; print("Imbalanced-Learn", imblearn.__version__)
```

Reply:
wouldn't `pip freeze` be much more comprehensive? Sometimes it's something annoying like, say, pytest, or jupyter, or who knows what :-)

Reply:
switched this to have show_versions